### PR TITLE
Interactive db:set

### PR DESCRIPTION
**Pull request by gulopine**
This moves database key and schema name back to being optional arguments, and if not supplied, they're requested interactively at the prompt. We fetch the list of available db keys and allow the user to select one, before asking for a schema name as plain text. It'll only ask for details that weren't provided in the command itself.

[asciicast](https://asciinema.org/a/8upn49zyw4yjj20xlz7kbbzmw)

### CODE REVIEW

File: lib/commands/connect/db-set.js [hunk 1]
```diff
@@ -2,36 +2,67 @@
...
+      {
+        name: 'db_key',
+        type: 'list',
+        message: "Select the config var that points to the database you'd like to use",
+        choices: yield fetchKeys(connection.app_name, context),
+        when: !context.flags.db
+      },
+      {
+        name: 'schema_name',
+        message: "Enter a schema name you'd like to use for the conneted data",
```
Comment (alouie-sfdc) [inline lib/commands/connect/db-set.js]: typo: connected

**Comment by sibson** [comment]
do we need a readme update?

**Comment by sibson** [comment]
:thumbsup: :shipit:

**Comment by gulopine** [comment]
README updated

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Heroku Connect Toolbelt Plugin
 
 # Commands
 
-    heroku connect:db:set                  - Configure a new connection
+    heroku connect:db:set                  - Set <data></data>atabase parameters
```
Comment (sibson) [inline README.md]: typo?

**Comment by sibson** [comment]
:thumbsup: :shipit:

**Pull request merged**